server/model/utils/json.go

```diff
@@ -1,7 +1,7 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
-package jsonutils
+package utils
```

Comment:
It doesn't feel right to dump this in utils. `utils`  is too broad for this. Why not keep `jsonutils`, or just an `errors` package?

Reply:
I'm somewhat worried about bikeshedding over this. This new `public` won't ship a v1 right away, and we'll have time to break the interface and relocate things to better places as needed.

But if we think having two new packages under `model/` (soon `public/`) with just a method or two each is better, I'm happy to make the changes!

Let me know either way, @agnivade and @wiggin77 :)

Reply:
Alright, don't want to delay this. But let's circle back on this sometime. :)